app/src/store/voice/call/index.ts

```diff
@@ -25,12 +24,9 @@ export const getCallInfo = (call: TwilioCall): CallInfo => {
   const to = call.getTo();
   const from = call.getFrom();
 
-  const initialConnectedTimestamp = match([
-    call.getInitialConnectedTimestamp(),
-    Platform.OS,
-  ])
-    .with([undefined, P._], () => undefined)
-    .with([P.not(undefined), P._], ([timestamp]) => Number(timestamp))
+  const initialConnectedTimestamp = match([call.getInitialConnectedTimestamp()])
```

Comment:
Another small change we should make here. We used to be matching on a tuple because we wanted to match on multiple values, but since it's just one now, we can get rid of the array wrapping the params here and in the pattern arms.